Dockerfile

```diff
@@ -1,30 +1,41 @@
-# setup build image
-FROM golang:1.11 AS build
+# Minify client side assets (JavaScript)
+FROM node:latest AS build-js
 
-# build Gophish binary
-WORKDIR /build/gophish
+WORKDIR /build
 COPY . .
-RUN go get -d -v ./...
-RUN go build
 
+RUN npm install gulp gulp-cli -g
+RUN npm install --only=dev
+RUN gulp
 
-# setup run image
-FROM debian:stable-slim
 
-RUN apt-get update && \
-    apt-get install --no-install-recommends -y \
-    jq && \
-    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+# Build Golang binary
+FROM golang:1.11 AS build-golang
+
+WORKDIR /go/src/github.com/gophish/gophish
+COPY . .
+
+RUN go get -v && go build -v
 
-# copy Gophish assets from the build image
-WORKDIR /gophish
-COPY --from=build /build/gophish/ /gophish/
-RUN chmod +x gophish
 
-# expose the admin port to the host
+# Runtime container
+FROM debian:stable-slim
+
+WORKDIR /opt/gophish
```

Comment:
When running this, I show that gophish is unable to open a SQLite database in /opt/gophish since the folder is owned by root.

It seems a fix for this would be to move this under the `useradd` command, as well as adding the `-m` flag to the `useradd` command in order to have the home directory created.